de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v2_0/writer/WriterNodeModel.java

```diff
@@ -525,25 +535,84 @@ private static void writeArchive(File archiveFile, FskPortObject portObject,
 
       final URI libUri = NodeUtils.getLibURI();
       List<String> missingPackages = new ArrayList<>();
-      List<VersionedPackage> versionedPackages = new ArrayList<>(portObject.packages.size());
-
-      // Get versions of R packages
-      for (String packageName : portObject.packages) {
-        String command = scriptHandler.getPackageVersionCommand(packageName);
-
-        try {
-          String packageVersion = scriptHandler.runScript(command, exec, true)[0];
-          versionedPackages.add(new VersionedPackage(packageName, packageVersion));
-
-          Path path = LibRegistry.instance().getPath(packageName);
-          if (path != null) {
-            File file = path.toFile();
-            archive.addEntry(file, file.getName(), libUri);
+      List<VersionedPackage> versionedPackages = new ArrayList<>();
+      String PythonDistributionVersion = null ;
+      if(portObject.modelMetadata != null && SwaggerUtil.getLanguageWrittenIn(portObject.modelMetadata) != null) {
+        String languageWrittenIn = SwaggerUtil.getLanguageWrittenIn(portObject.modelMetadata);
+        String pythonPath = null; // Path to the Python executable in the conda environment
+        String envPath = null; // Path to the conda environment
+        if(PreferencePage.getPython3CondaEnvironmentDirectoryPath() != null)
+          envPath = PreferenceInitializer.getPython3Env();
+        else
+          envPath = PreferenceInitializer.getPython2Env();
+        
+        pythonPath =envPath+"/bin/python";
```

Comment:
Same problem as in FSKEditorJSNodemModel.java (line 544), in Windows it doesn't work.